src/app/proposal/proposal-list/proposal-list.component.html

```diff
@@ -1,19 +1,24 @@
-
-<div class="card mb-1" *ngFor="let proposal of proposals">
-  <div class="card-block row m-1 col-12">
+<div class="proposal-list-container">
+  <div class="card proposal-card mb-3" *ngFor="let proposal of proposals">
     <div class="card-body row align-items-center">
-      <div class="col-6 mt-1">
-        <h6 class="card-subtitle text-muted">Title</h6>
-        <a class="card-text" [routerLink]="['/proposals', proposal.id]">{{proposal.title}}</a>
+      <div class="col-6">
+        <h6 class="card-subtitle">Title</h6>
+        <a class="card-text" [routerLink]="['/proposals', proposal.id]">
+          {{ proposal.title }}
+        </a>
       </div>
-      <div class="col-6 mt-1">
-        <h6 class="card-subtitle text-muted">Owner</h6>
-        <p class="card-text">{{proposal.owner}}</p>
+      <div class="col-6">
+        <h6 class="card-subtitle">Owner</h6>
+        <p class="card-text">{{ proposal.owner }}</p>
```

Comment:
The proposal owner is linked from the proposal. To display its username, you should first retrieve the linked resource (the user) following the corresponding relation (for instance `/proposals/1/owner`). Using the HateoasClient, this is facilitated by the corresponding function `getRelation`in the retrieved resource: https://github.com/lagoshny/ngx-hateoas-client?tab=readme-ov-file#GetRelation